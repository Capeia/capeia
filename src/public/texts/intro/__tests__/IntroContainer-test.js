// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<IntroContainer>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/intro/IntroContainer')
    const Component = require('public/texts/intro/IntroContainer').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
