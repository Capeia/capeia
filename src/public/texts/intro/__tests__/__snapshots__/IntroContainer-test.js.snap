// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroContainer> renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="IntroContainer row"
  >
    <div
      className="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1"
    />
  </div>
</div>
`;
